pallet-logion-loc/src/lib.rs

```diff
@@ -974,6 +989,30 @@ pub mod pallet {
 
             Ok(().into())
         }
+
+        /// Creates a new Identity LOC whose requester is an Ethereum address.
```

Comment:
Actually, requester is an "other" account and currently, only Ethereum addresses are supported.

Reply:
Correct. Comment adapted.